src/components/v2/MyAccount/MyAccount.tsx

```diff
@@ -0,0 +1,80 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { ISliderProps, Slider } from '../Slider';
+import { useMyAccountStyles as useStyles } from './MyAccountStyles';
+import { Toggle } from '../Toggle';
+import { Icon } from '../Icon';
+
+interface IMyAccountProps {
+  netApy: string;
+  dailyEarnings: string;
+  supplyBalance: string;
+  borrowBalance: string;
+  borrowLimit: string;
+  borrowLimitSum: string;
+  safeLimit: string;
+  sliderProps: ISliderProps;
+}
+
+export const MyAccount = ({
+  netApy,
+  dailyEarnings,
+  supplyBalance,
+  borrowBalance,
+  borrowLimit,
+  borrowLimitSum,
+  safeLimit,
+  sliderProps,
+}: IMyAccountProps) => {
+  const [isToggleSwitched, setToggleSwitched] = useState(true);
+  const handleSwitch = (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => {
+    setToggleSwitched(checked);
+  };
+
+  const styles = useStyles();
+  return (
+    <Box css={styles.root} component={Paper}>
+      <div css={styles.row}>
+        My account
+        <span css={styles.apyWithXvs}>
+          <Icon css={styles.iconInfo} name="info" /> APY with XVS
+          <Toggle cssStyles={styles.toggle} value={isToggleSwitched} onChange={handleSwitch} />
+        </span>
+      </div>
+      <div>
+        <p css={[styles.label, styles.netApyLabel]}>
+          Net APY <Icon css={styles.iconInfo} name="info" />
+        </p>
+        <span css={styles.netApy}>{netApy}</span>
+      </div>
+      <ul css={styles.list}>
+        <li css={styles.item}>
+          <p css={[styles.label, styles.labelListItem]}>Daily earnings</p>
```

Comment:
how do you think, guys @coreyar @therealemjy, should we move texts in separate json config for translating feature or just resources organizing?